concrete/src/Area/Layout/Layout.php

```diff
@@ -58,7 +58,7 @@ public static function getByID($arLayoutID)
         $al = null;
         $db = Database::connection();
         $row = $db->GetRow('select arLayoutID, arLayoutIsPreset, arLayoutUsesThemeGridFramework from AreaLayouts where arLayoutID = ?', [$arLayoutID]);
-        if (is_array($row) && $row['arLayoutID']) {
+        if (is_array($row) && array_key_exists('arLayoutID', $row) && $row['arLayoutID']) {
```

Comment:
We can simply replace `$db->GetRow()` with `$db->fetchAssociative()` (which returns `false` is there's no row found).

Reply:
971f15533ed9eaed8b6b0b2d55c4adfbcf5489bb